aep/general/0135/aep.md.j2

```diff
@@ -0,0 +1,223 @@
+# Delete
+
+In REST APIs, it is customary to make a `DELETE` request to a resource's URI
+(for example, `/v1/publishers/{publisher}/books/{book}`) in order to delete
+that resource.
+
+Resource-oriented design ([AEP-121][aep-121]) honors this pattern through the
+`Delete` method. This method accepts the URI representing that resource and
+usually returns an empty response.
+
+## Guidance
+
+APIs **should** generally provide a delete method for resources unless it is
+not valuable for users to do so.
+
+The Delete method **should** succeed if and only if a resource was present and
+was successfully deleted. If the resource did not exist, the method **should**
+send a `404 Not found` (`NOT_FOUND`) error.
+
+### Requests
+
+{% tab oas -%}
+
+Delete methods are specified using the following pattern:
+
+{% sample 'delete.oas.yaml', 'paths' %}
+
+- The HTTP verb **must** be `DELETE`.
+- There **must not** be a request body in API description.
+- If a delete request contains a body, the body **must** be ignored, and **must
+  not** cause an error (this is required by [RFC 9110][])
+- The request **must not** require any fields in the query string. The request
+  **should not** include optional fields in the query string unless described
+  in another AEP.
+- Delete methods **should** return `204 No Content` with no response body, or
+  `202 Accepted` with a representation of the operation in the response body
+  if the delete is [long-running](#long-running-delete).
+- If the API is operating on the [Management Plane][], the method should have
```

Comment:
nit: this guidance is common across OAS and protobuf. Maybe shared guidance belongs on the top before the protocol-specific guidance?

Reply:
Agreed - that's the pattern I've been following for my recent PRs.

Reply:
Moved to the common section.